vlib/v/parser/tests/struct_anon_invalid_embed_err.out

```diff
@@ -1,4 +1,4 @@
-vlib/v/parser/tests/struct_anon_invalid_embed_err.vv:2:16: error: invalid anonymous struct
+vlib/v/parser/tests/struct_anon_invalid_embed_err.vv:2:16: error: cannot embed non-struct `Int`
     1 | type Int = int
     2 | type MyIndex = struct {
       |                ~~~~~~~~
```

Comment:
It still underlines `struct {`. Can it underline the field that was not a struct instead?

Reply:
Done